### DISCUSSION TITLE
테스터 관리 인원에 등록이 되있는데 접속이 되지 않습니다.

### DISCUSSION THREAD

**Discussion by devinhyeok** [naverz/zepeto-studio-kor#255] [5. Q&A]
테스터에 인원을 등록하고 접속하게 했는데  테스터로 등록되지 않았다고 뜹니다.
업데이트까지 시간이 걸리나요?

**Comment by cheolhui-lee** [comment]
안녕하세요.
캐쉬등으로 인해 실제 반영에는 시간이 소요될 수 있습니다.
혹시 수시간이 지났음에도 정상 동작을 하지 않는다면 말씀 부탁드립니다.
